package.json

```diff
@@ -8,6 +8,7 @@
   "scripts": {
     "test-selenium": "./tools/execute-selenium-tests",
     "test-integration": "npm run clean_db && ./node_modules/.bin/mocha test/integration/test/*.js",
+    "test-integration-travis": "npm run clean_db && ./node_modules/.bin/mocha test/integration/test/travis_*.js",
```

Comment:
I think we should execute the tests by default on travis, and only exclude the specific ones.